src/content/docs/apis/intro-apis/new-relic-api-keys.mdx

```diff
@@ -174,11 +173,9 @@ Basic information about our keys are in the [API keys overview table](#overview-
   >
 Our main key used for data ingest is called the license key. In the API keys UI and in [NerdGraph](/docs/apis/nerdgraph/examples/use-nerdgraph-manage-license-keys-user-keys), this key is sometimes referenced as `ingest - license`. 
 
-The license key is required for almost all New Relic data ingest. The exceptions are browser monitoring data (which uses a [browser key](#overview-keys)) and mobile monitoring data (which uses a [mobile app token](#overview-keys)). 
-
-The license key is a 40-character hexadecimal string associated with a New Relic account. When you first [sign up](/docs/subscriptions/creating-your-new-relic-account) for New Relic, that creates an organization with a single account, and that account has its own license key. If more accounts are added, each account starts with its own license key. The license key originally created for an account cannot be deleted but you can create additional license keys that can be managed and deleted, and this is useful for implementing [security policies](#security-practices) (for example, key rotation). If you need to delete an account's original license key, [contact support](https://support.newrelic.com).
+The license key is required for almost all New Relic data ingest. The exceptions are browser monitoring data (which uses a browser key) and mobile monitoring data (which uses a mobile app token). 
 
-For security-related best practices, see [Best practices](#security-practices).
+The license key is a 40-character hexadecimal string associated with a New Relic account. When you first [sign up](/docs/subscriptions/creating-your-new-relic-account) for New Relic, that creates an organization with a single account, and that account has its own license key. If more accounts are added, each account starts with its own license key. The license key originally created for an account cannot be deleted but you can create additional license keys that can be managed and deleted, and this is useful for implementing security-practices such as key rotation. If you need to delete an account's original license key, [contact support](https://support.newrelic.com).
```

Comment:
Suggestion: When you first [sign up](/docs/subscriptions/creating-your-new-relic-account) for New Relic, an organization with a single account and its own license key are created.